tests/html/chartAuthoring-Tests.html

```diff
@@ -30,6 +30,7 @@
         <script src="../../src/js/dataEntry.js"></script>
         <script src="../../src/js/dataEntryPanel.js"></script>
         <script type="text/javascript" src="../../src/js/d3ViewComponent.js"></script>
+        <script src="../../src/js/svgDrawingArea.js"></script>
```

Comment:
There are two ways to include .js files in this project, one is with ```type="text/javascript"``` and one without. Can we sync up to pick one way? Thanks.